### ISSUE TITLE
state vs. trait

### ISSUE THREAD

**Issue by TEITechnicalCouncil** [TEIC/TEI#843]
I notice that both &lt;state&gt; and &lt;trait&gt; can have datable attributes on them even though &lt;trait&gt; is meant for properties that are "in principle" unchanging.  So at first it would seem that the datable attributes should not be allowed on &lt;trait&gt; unless we allow that certain traits may change but are still to be considered traits and not states.  Is the intention?

Further confusion comes from the example given in the Guidelines for &lt;trait&gt;, involving citizenship that lasted for only a certain period.  It seems to me that this example is more appropriate for the &lt;state&gt; element, unless this is an example of one of these traits not to be considered a state.

In short, a bit of clarification on the use of these elements would be good here.

Original comment by: sf_user_*anonymous
